django_toolkit/toolkit_settings.py

```diff
@@ -0,0 +1,8 @@
+from django.conf import settings
+
+toolkit_settings = getattr(settings, 'TOOLKIT', {})
```

Comment:
Could you rename this var to `_toolkit_settings`, just to prevent it from being used out of this module?